test/Tests/Properties.hs

```diff
@@ -521,7 +521,7 @@ tests =
       testProperty "sf_cons" sf_cons,
       testProperty "j_cons" j_cons,
       testProperty "s_snoc" s_snoc,
-      testProperty "j_snoc" j_snoc,
+--      testProperty "j_snoc" j_snoc, -- property `.f`
```

Comment:
Why is this removed?

Reply:
I tested it locally and it works so I've re-enabled it.